<launch>
    <include file="$(find harmoni_face)/launch/face_service_misty.launch">
        <!--<arg name="test_id" value="default" />-->
    </include>
    <include file="$(find harmoni_tts)/launch/tts_service.launch">
        <!--arg name="test_id" value="default" />-->
    </include>
    <include file="$(find harmoni_web)/launch/web_service.launch">
        <!--<arg name="test_id" value="default" />-->
    </include>
    <!--<include file="$(find harmoni_speaker)/launch/speaker_service_misty.launch">
        <arg name="test_id" value="default" />
    </include>
    <include file="$(find harmoni_gesture)/launch/gesture_service_misty.launch">
        <arg name="test_id" value="default" />
    </include>-->
</launch>